Test/ParallelPrimitives/RadixSortKernels.h

```diff
@@ -569,8 +547,121 @@ __device__ void localSort4bitMulti( int* keys, u32* ldsKeys, const int START_BIT
 		keys[i] = ldsKeys[threadIdx.x * N_ITEMS_PER_WI + i];
 	}
 }
-
 __device__ void localSort8bitMulti( int* keys, u32* ldsKeys, const int START_BIT )
+{
+	__shared__ unsigned table[BIN_SIZE];
+
+	for( int i = threadIdx.x; i < BIN_SIZE; i += SORT_WG_SIZE )
+	{
+		table[i] = 0U;
+	}
+
+	LDS_BARRIER;
+
+	for( int i = 0; i < SORT_N_ITEMS_PER_WI; ++i )
+	{
+		const int tableIdx = ( keys[i] >> START_BIT ) & RADIX_MASK;
+		atomicAdd( &table[tableIdx], 1 );
+	}
+
+	LDS_BARRIER;
+
+	int globalSum = 0;
+	for( int binId = 0; binId < BIN_SIZE; binId += WG_SIZE * 2 )
```

Comment:
Shouldn't this be `SORT_WG_SIZE  * 2` instead?

Reply:
fixed